### DISCUSSION TITLE
Ability to search by IMDB ID using Graphql?

### DISCUSSION THREAD

**Discussion by mkcfdc** [bitmagnet-io/bitmagnet#354] [Q&A]
Hello,

I've been searching through the docs but don't see a way to search directly via IMDB ID using the Graphql API. I know using the SearchQuery does work, but is very slow when done on a database of over 5 million.

Just wondering if I'm missing something, or maybe it's just not possible.

Thanks!

**Comment by ralyodio** [comment]
is there a way to get imdb_id from bitmagnet indexing?? I can't find anything. if its in the db i can use it. but i don't see any ids being recorded for any kind of metadata provider.

**Comment by rraymondgh** [comment]
to extract using graphql:

```
{
  torrentContent {
    search(
      input: { queryString: "the snowman", facets: { contentType: { filter: [movie] } } }
    ) {
      items {
        title
        content {
          source
          id
          attributes {
            source
            key
            value
          }
        }
      }
    }
  }
}
```

to extract using sql:

```
SELECT
    t.name,
    tc.content_source,
    tc.content_id,
    ca.source,
    ca.key,
    ca.value
FROM
    torrents t,
    torrent_contents tc,
    content_attributes ca
WHERE
    t.name LIKE '%snowman%'
    AND t.info_hash = tc.info_hash
    AND tc.content_id = ca.content_id
    AND tc.content_source = ca.content_source
    AND ca.source = 'imdb'
LIMIT 10;
```

**Reply by ralyodio** [reply]
I get zero results using the sql

**Reply by ralyodio** [reply]
Ok so I figured out the content_attributes.source all have tmdb for me   Is there anyway to convert that to an IMDb id?

**Comment by ralyodio** [comment]
I get "Success. No rows returned" when running that sql

**Reply by rraymondgh** [reply]
> I get "Success. No rows returned" when running that sql

probably means you have no torrents matching the wild carded name.  could use `t.name LIKE '%'`

**Reply by ralyodio** [reply]
My source is tmdb not imdb is there anyway to convert that to and imdb id?
